vhdbuilder/release-notes/autonotes/main.go

```diff
@@ -212,22 +212,24 @@ var defaultPath = filepath.Join("vhdbuilder", "release-notes")
-	"1804-gen2-containerd":          filepath.Join("AKSUbuntu", "gen2", "1804containerd"),
-	"1804-gpu-containerd":           filepath.Join("AKSUbuntu", "gen1", "1804gpucontainerd"),
-	"1804-gen2-gpu-containerd":      filepath.Join("AKSUbuntu", "gen2", "1804gpucontainerd"),
-	"1804-fips-containerd":          filepath.Join("AKSUbuntu", "gen1", "1804fipscontainerd"),
-	"1804-fips-gen2-containerd":     filepath.Join("AKSUbuntu", "gen2", "1804fipscontainerd"),
-	"1804-fips-gpu-containerd":      filepath.Join("AKSUbuntu", "gen1", "1804fipsgpucontainerd"),
-	"1804-fips-gen2-gpu-containerd": filepath.Join("AKSUbuntu", "gen2", "1804fipsgpucontainerd"),
-	"marinerv1":                     filepath.Join("AKSCBLMariner", "gen1"),
-	"marinerv1-gen2":                filepath.Join("AKSCBLMariner", "gen2"),
-	"marinerv2-gen2":                filepath.Join("AKSCBLMarinerV2", "gen2"),
-	"2004-cvm-gen2-containerd":      filepath.Join("AKSUbuntu", "gen2", "2004cvmcontainerd"),
-	"2204-containerd":               filepath.Join("AKSUbuntu", "gen1", "2204containerd"),
-	"2204-gen2-containerd":          filepath.Join("AKSUbuntu", "gen2", "2204containerd"),
+	"1804-containerd":                   filepath.Join("AKSUbuntu", "gen1", "1804containerd"),
+	"1804-gen2-containerd":              filepath.Join("AKSUbuntu", "gen2", "1804containerd"),
+	"1804-gpu-containerd":               filepath.Join("AKSUbuntu", "gen1", "1804gpucontainerd"),
+	"1804-gen2-gpu-containerd":          filepath.Join("AKSUbuntu", "gen2", "1804gpucontainerd"),
+	"1804-fips-containerd":              filepath.Join("AKSUbuntu", "gen1", "1804fipscontainerd"),
+	"1804-fips-gen2-containerd":         filepath.Join("AKSUbuntu", "gen2", "1804fipscontainerd"),
+	"1804-fips-gpu-containerd":          filepath.Join("AKSUbuntu", "gen1", "1804fipsgpucontainerd"),
+	"1804-fips-gen2-gpu-containerd":     filepath.Join("AKSUbuntu", "gen2", "1804fipsgpucontainerd"),
+	"marinerv1":                         filepath.Join("AKSCBLMariner", "gen1"),
+	"marinerv1-gen2":                    filepath.Join("AKSCBLMariner", "gen2"),
+	"marinerv2-gen2":                    filepath.Join("AKSCBLMarinerV2", "gen2"),
+	"marinerv2-gen2-kata":               filepath.Join("AKSCBLMarinerV2", "gen2kata"),
+	"marinerv2-gen2-arm64":              filepath.Join("AKSCBLMarinerV2", "gen2arm64"),
+	"marinerv2-gen2-trustedlaunch":      filepath.Join("AKSCBLMarinerV2", "gen2tl"),
+	"marinerv2-gen2-kata-trustedlaunch": filepath.Join("AKSCBLMarinerV2", "gen2katatl"),
+	"2004-cvm-gen2-containerd":          filepath.Join("AKSUbuntu", "gen2", "2004cvmcontainerd"),
+	"2204-containerd":                   filepath.Join("AKSUbuntu", "gen1", "2204containerd"),
+	"2204-gen2-containerd":              filepath.Join("AKSUbuntu", "gen2", "2204containerd"),
+	"2204-arm64-gen2-containerd":        filepath.Join("AKSUbuntu", "gen2", "2204arm64containerd"),
+	"2204-tl-gen2-containerd":           filepath.Join("AKSUbuntu", "gen2", "2204tlcontainerd"),
```

Comment:
inconsistency in tl vs trustedlaunch for mariner -__- just mentally bookmarking ways we've made this annoying to automate....

not an issue for this.